dbt/adapters/trino/connections.py

```diff
@@ -144,15 +144,14 @@ def exception_handler(self, sql):
             logger.debug(exc)
             raise dbt.exceptions.RuntimeException(str(exc))
 
+    # For connection in auto-commit mode there is no need to start
+    # separate transaction. If using auto-commit, the client
+    # will create a new transaction for each query
     def add_begin_query(self):
-        connection = self.get_thread_connection()
-        with self.exception_handler("handle.start_transaction()"):
-            connection.handle.start_transaction()
+        logger.debug("NotImplemented: add_begin_query")
```

Comment:
The `debug` log statement does not feel right. A future maintainer may think that this method requires implementation when it actually doesn't.

Reply:
Agreed. Can it be a comment with the method body just calling `pass`?

Reply:
Done!